digdag-docs/src/profiler.md

```diff
@@ -0,0 +1,89 @@
+
+# Digdag profiler (experimental)
+
+Digdag user may sometimes want to know how much attempts and tasks delayed in Digdag. Digdag has an experimental profiler for that.
+
+## How to use
+
+Current profiler requires Digdag source code as follows
+
+```
+$ git clone https://github.com/treasure-data/digdag.git
+$ cd digdag
+$ ./gradlew digdag-profiler:installDist
+$ digdag-profiler/build/install/digdag-profiler/bin/digdag-profiler -c $HOME/.config/digdag/digdag.properties --from 2020-12-30T00:00:00
+{"attempts":2048,"totalTasks":8192,"totalRunTasks":8190,"totalSuccessTasks":8189,"totalErrorTasks":1,"mostDelayedTask":{"subtaskConfig":{},"exportParams":{},"resetStoreParams":[],"storeParams":{},"report":{"inputs":[],"outputs":[]},"error":{},"resumingTaskId":null,"id":369,"attemptId":1024,"upstreams":[],"updatedAt":"2021-02-16T08:56:01Z","retryAt":null,"startedAt":"2021-02-16T08:56:00Z","stateParams":{},"retryCount":0,"parentId":1000,"fullName":"+echo^failure-alert","config":{"local":{"_type":"notify","_command":"Workflow session attempt failed"},"export":{}},"taskType":0,"state":"success","stateFlags":0},"startDelayMillis":{"min":0,"max":2000,"average":67,"stddev":286},"execDurationMills":{"min":0,"max":11000,"average":1115,"stddev":1546}}
+```
+
```

Comment:
Is thre any reason, you don't integrate `profiler` to `digdag` execute binary ?
If we can use the profiler as `digdag profiler <option>`, it may be convenient and natural for digdag manner.

Reply:
The reasons why I created this tool separately are:
1. this tool is not for normal Digdag users, but for kind of Digdag developers
2. it's a bit uncommon since it accesses the database directly not thought REST API
3. I'm sure the existing all-in-one monolithic executable cli is the best way

But, on second thought, probably #2 doesn't matter since `digdag migrate` already exists. And #1 and #3 may be a matter of taste. So I think we can move this tool into the existing executable cil. Do you think we'd better merge this functionality to the existing cli?

Reply:
As we talked later, I agree with your idea. I'll update this PR. Thanks!